lib/user.rb

```diff
@@ -0,0 +1,42 @@
+class User 
+  def self.create(first_name, last_name, username, email, password)
+    current_user = DatabaseConnection.query(
+      "INSERT INTO users (first_name, last_name, username, email, password)
+       VALUES('#{first_name}', '#{last_name}', '#{username}', '#{email}', '#{password}')
+       RETURNING *"
+      )
+    User.get_current(current_user.first['id'])
+  end
+
+  def self.get_current(id) 
+    rs = DatabaseConnection.query("SELECT * FROM users WHERE id = '#{id}'")
+    result = rs.first
+    User.new(result['first_name'], result['last_name'], result['username'], result['email'], result['id'])
+  end
+
+  def self.logout
+    nil
+  end
+
+  def self.login(email, password)
+    result = DatabaseConnection.query(
+      "SELECT * FROM users WHERE email = '#{email}' AND password = '#{password}'"
```

Comment:
you shouldn't store passwords in plain text. you could use bcrypt to create a hash of the password to store, and to check the hash of a given password against the hash of a stored password